ibm/service/power/resource_ibm_pi_instance.go

```diff
@@ -944,12 +953,76 @@ func isPIInstanceRefreshFunc(client *st.IBMPIInstanceClient, id, instanceReadySt
 	}
 }
 
+func isWaitForPIInstancePlacementGroupAdd(ctx context.Context, client *st.IBMPIPlacementGroupClient, pgID string, id string) (interface{}, error) {
+	log.Printf("Waiting for PIInstance Placement Group (%s) to be updated ", id)
+
+	queryTimeOut := activeTimeOut
+
+	stateConf := &retry.StateChangeConf{
+		Pending:    []string{State_Adding},
+		Target:     []string{State_Added},
+		Refresh:    isPIInstancePlacementGroupAddRefreshFunc(client, pgID, id),
+		Delay:      30 * time.Second,
+		MinTimeout: queryTimeOut,
+		Timeout:    120 * time.Minute,
```

Comment:
Don't you think 2 hrs is too much for adding/deleting PG?

Reply:
Yea that's a good point. Should lower that to like 10 minutes.